app/scripts/controllers/console.js

```diff
@@ -79,6 +84,43 @@ angular.module('console', ['sql'])
           });
         };
 
+        $scope.isGeopoint = function isGeopoint(dataType) {
```

Comment:
All these checking functions can be moved to a separate service which will make them reusable and a lot easier to test